bazel-dre-1

```diff
@@ -0,0 +1 @@
+/private/var/tmp/_bazel_pietro.di.marco/b166ba5137d2b5b80ccec0055091def4/execroot/dre
```

Comment:
Is this needed or is it a leftover?

Reply:
eah that's a leftover let me get rid of it